### ISSUE TITLE
Ability to set browser for "Experimental Live Preview"

### ISSUE THREAD

**Issue by kevinbe71** [adobe/brackets#10714]
I currently keep "Metro" IE (Windows 8.1) as my default browser because setting a different browser disables the "Metro" capability.  However, I want to use Firefox for experimental live preview.  It seems as if the current implementation doesn't allow any configuration of the browser it uses... it just assumes you want to use the default browser!  This is also problematic because "Metro" IE doesn't allow loopback so it is impossible to use for experimental live preview.

**Comment by ApfelUser** [comment]
:+1:

**Comment by redmunds** [comment]
@kevinbe71 Yes, that UI will eventually be improved. For now, just copy the URL from your default browser and paste it into any other browser.

**Comment by Denisov21** [comment]
:+1:
